src/include/parser/parser.h

```diff
@@ -92,6 +92,9 @@ extern bool pltsql_case_insensitive_identifiers;
 
 extern char* pltsql_server_collation_name;
 
+/* Dump and Restore */
+extern bool restore_tsql_datetime2;
```

Comment:
We can use get_config_option function instead of defining an extern object here.